### ISSUE TITLE
Biospec - Dshbrd - Biohazard Bag Mouthwash Full Specimen ID not saved

### ISSUE THREAD

**Issue by wyattks** [episphere/connect#425]
When testing adding a tube after the rest of the collection was moved forward to shipping, after entering in the ID for the tube and clicking "Save and Continue" I received an error stating the Full Specimen ID for the Mouthwash Biohazard Bag needs to be entered.   But the Edit button is no longer active and I had previously entered the ID.  The bag/ID shows up on the shipping dashboard as well.  This is happening for Participant "Tdrp", Collection ID: CXA963963.

**Comment by wyattks** [comment]
I just tried again and it seems to be happening the first time you hit save.  It is not saving that last ID entered.  I did a scenario where I entered all tubes except the SST tubes.  I saved everything and went back to the search screen.  I searched using the Collection ID, entered in the SST tube information.  I saw at that point the Mouthwash biohazard bag ID I had previously entered was not saved but because I have not completed my review yet the Edit button was still present.  I entered in the information again, saved and returned to the search.  When I searched by the collection ID and returned to the page, the mouthwash biohazard Bag ID was present.
